osu.Game/Screens/OnlinePlay/Lounge/Components/DrawableRoom.cs

```diff
@@ -112,114 +133,209 @@ public DrawableRoom(Room room)
+                    Children = new Drawable[]
+                    {
+                        new Box
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                            Colour = Color4Extensions.FromHex(@"#27302E"),
+                        },
+                        new OnlinePlayBackgroundSprite(BeatmapSetCoverType.List)
+                        {
+                            RelativeSizeAxes = Axes.Both
+                        },
+                    }
                 },
                 new Container
                 {
+                    Name = @"Room content",
                     RelativeSizeAxes = Axes.Both,
-                    Padding = new MarginPadding(SELECTION_BORDER_WIDTH),
+                    Padding = new MarginPadding { Left = 20, Vertical = -0.5f },
                     Child = new Container
                     {
                         RelativeSizeAxes = Axes.Both,
                         Masking = true,
                         CornerRadius = corner_radius,
-                        EdgeEffect = new EdgeEffectParameters
-                        {
-                            Type = EdgeEffectType.Shadow,
-                            Colour = Color4.Black.Opacity(40),
-                            Radius = 5,
-                        },
                         Children = new Drawable[]
                         {
-                            new Box
+                            // This resolves internal 1px gaps due to applying the (parenting) corner radius and masking across multiple filling background sprites.
+                            new BufferedContainer
                             {
                                 RelativeSizeAxes = Axes.Both,
-                                Colour = Color4Extensions.FromHex(@"212121"),
-                            },
-                            new StatusColouredContainer(transition_duration)
-                            {
-                                RelativeSizeAxes = Axes.Y,
-                                Width = stripWidth,
-                                Child = new Box { RelativeSizeAxes = Axes.Both }
-                            },
-                            new Container
-                            {
-                                RelativeSizeAxes = Axes.Y,
-                                Width = cover_width,
-                                Masking = true,
-                                Margin = new MarginPadding { Left = stripWidth },
-                                Child = new OnlinePlayBackgroundSprite(BeatmapSetCoverType.List) { RelativeSizeAxes = Axes.Both }
+                                Children = new Drawable[]
+                                {
+                                    new Box
+                                    {
+                                        RelativeSizeAxes = Axes.Both,
+                                        Colour = Color4Extensions.FromHex(@"#27302E"),
```

Comment:
Same colour used in a few places, split out to a named variable (is this a special colour not in the scheme?)

Reply:
Dunno where this colour comes from. It appears somewhere inbetween the blue and green colour schemes - it's more like a cyan hue?

@arflyte Where does the background colour of these multiplayer panels come from?
![image](https://user-images.githubusercontent.com/1329837/128349115-79626d6c-5292-4448-b2f7-2f5af01d136c.png)

Reply:
FWIW it seems *somewhat* similar to `B5`'s HSL:
- this – `#27302E`: `167°, 10%, 17%`
- `B5` – `#222A28`: `160°, 10%, 15%`